src/enclave/enclave_oe.c

```diff
@@ -24,30 +20,56 @@ int __sgx_init_enclave()
     return __libc_init_enclave(sgxlkl_enclave->argc, sgxlkl_enclave->argv);
 }
 
-void sgxlkl_enclave_show_attribute(const void* sgxlkl_enclave_base)
+#ifdef DEBUG
+static void _size_uint64_to_str(uint64_t size, char* buf, uint64_t len)
+{
+    int i = 0;
+    double bytes = size;
+    const char* units[] = {"B", "KB", "MB", "GB", "TB", "PB"};
```

Comment:
```suggestion
    const char* units[] = {"B", "KiB", "MiB", "GiB", "TiB", "PiB", "EiB"};
```
2^64 B is 16 EiB, so we need EiB to support all possible values in callers of this.

Reply:
Done.